app/models/role.rb

```diff
@@ -0,0 +1,26 @@
+class Role < ActiveRecord::Base
+  has_many :auditions
+
+    def role_auditions
+      Audition.where(role_id: id)
+    end
```

Comment:
Same with this one. not needed we should get it out of the box with associations